.github/workflows/reusable-docs.yml

```diff
@@ -100,7 +100,7 @@ jobs:
     - name: 'Configure CPython'
       run: ./configure --with-pydebug
     - name: 'Build CPython'
-      run: make -j4
+      run: make all-with-ensurepip-dists-bundled -j4
     - name: 'Install build dependencies'
       run: make -C Doc/ PYTHON=../python venv
```

Comment:
@AA-Turner so I looked into whether bundling pip into ensurepip is needed in doctests and the answer is “yes”. This line calls `../python -m venv` (so it uses the Python build under test and not some OS-provided copy). The `venv` module relies on `ensurepip` to provision its `site-packages/`. When there's no pip wheel, it crashes (as it should!).